addons/stock/models/stock_orderpoint.py

```diff
@@ -137,14 +138,14 @@ def _compute_json_popover(self):
                 'title': _('Replenishment'),
                 'icon': 'fa-area-chart',
                 'popoverTemplate': 'stock.leadDaysPopOver',
-                'lead_days_date': fields.Date.to_string(orderpoint.lead_days_date),
+                'lead_days_date': format_date(self.env, orderpoint.lead_days_date),
                 'lead_days_description': lead_days_description,
-                'today': fields.Date.to_string(fields.Date.today()),
+                'today': format_date(self.env, fields.Date.today()),
                 'trigger': orderpoint.trigger,
-                'qty_forecast': orderpoint.qty_forecast,
-                'qty_to_order': orderpoint.qty_to_order,
-                'product_min_qty': orderpoint.product_min_qty,
-                'product_max_qty': orderpoint.product_max_qty,
+                'qty_forecast': FloatConverter.value_to_html(orderpoint.qty_forecast, {'decimal_precision': 'Product Unit of Measure'}),
```

Comment:
I would use float_repr instead

Reply:
because `float_repr` doesn't format in the language:
float_repr(2000.5, 2) -->  2000.50

`python`: `2000.5`
`US` : `2.000,50`
`Fr` : `2 000,50`